Blish HUD/GameServices/ArcDps/SocketListener/RawCombatEventArgs.cs

```diff
@@ -7,7 +7,13 @@ public class RawCombatEventArgs : EventArgs {
 
         public enum CombatEventType {
 
+            /// <summary>
+            /// ArcDps calculations and display values.
+            /// </summary>
             Area,
+            /// <summary>
+            /// Local recording of just the own character similar to the in-game combat log in the chat window.
```

Comment:
> Local recording of just the own character

This is oddly worded.  Is it meant to say "Local recording of just _your_ own character..."

Otherwise, everything else looks great.  👍

Reply:
Fixed and added some missing doc.